project_setup/scripts/iam-to-github

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env python
+
+"""Create GitHub secrets from AWS credentials extracted from Terraform state.
+
+This command must be executed in the directory containing the .terraform state
+within a GitHub project.  It will attempt to detect the repository name from
+the projects git origin.
```

Comment:
It would be nice to clarify that you want the `.terraform` directory (which stores the Terraform state) for people just getting into the ecosystem. Also a small nitpick, but would you add an apostrophe in "projects".

Reply:
Fixed in: 3a6a031

Reply:
Thanks!